virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -71,6 +71,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-29987: Move instancetype API from v1alpha2 to v1beta1. As of 01/03/2024, this is an In Progress TP RN for 4.14 (CNV-21920).
 
 //CNV-31433: virtio-serial channel for SAP
+* Admins can now expose a limited set of host and virtual machine (VM) metrics to a guest VM through a `virtio-serial` port by enabling the `downwardMetrics` feature gate in the `HyperConverged` custom resource (CR), and adding the spec `downwardMetrics`. These metrics can be accessed by using the `vm-dump-metrics` tool.
```

Comment:
`vm-dump-metrics` tool is not available in RHEL9. Do you want to officially support it? Can you please check if it is available in 4.15 downstream documentation?

Reply:
cc @jcanocan

Reply:
I checked and it's there the doc explaining downwardMetrics is there.
But a comment about RHEL9 not supported should exist in my opinion

Reply:
There's the chance to use sequent of commands to fetch the downwardMetrics instead of using the `vm-dump-metrics` tool:

```bash
$ sudo sh -c 'printf "GET /metrics/XML\n\n" > /dev/virtio-ports/org.github.vhostmd.1'
$ sudo cat /dev/virtio-ports/org.github.vhostmd.1
```

Reply:
@jcanocan we don't have a page about downwardmetrics in 4.15 documentation about how to start, featuregate to enable/disable, how to configure. if we have in release notes but downstream docs don't talk about supporting it ,it will not look good from user prespective

Reply:
It is too late to add the downstream documentation prior to the release on Feb 28th. The work is planned to be completed shortly after the 4.15 release and here is the Jira story https://issues.redhat.com/browse/CNV-27866. Unfortunately, the docs team has capacity constraints and the team concentrated on the highest priority items first.

Reply:
cc @dominikholler  wdyt

Reply:
> There's the chance to use sequent of commands to fetch the downwardMetrics instead of using the `vm-dump-metrics` tool:
> 
> ```shell
> $ sudo sh -c 'printf "GET /metrics/XML\n\n" > /dev/virtio-ports/org.github.vhostmd.1'
> $ sudo cat /dev/virtio-ports/org.github.vhostmd.1
> ```

@jcanocan doesn't look good if this goes in release docs. should be part of real documentation